src/earn/prepareTransactions.ts

```diff
@@ -64,6 +70,7 @@ export async function prepareDepositTransactions({
       shortcutId,
       ...args,
       ...pool.shortcutTriggerArgs?.[shortcutId],
+      enableSwapFee,
```

Comment:
I guess the triggerShortcut call will discard this for deposit, but to be clean, you can set this conditionally in args for swap-deposit, right next to swapFromToken

Reply:
Fixed in 5140644529608dc9fe9a86f5d4ecca862232562c!